tests/utils/pvc.go

```diff
@@ -154,6 +155,14 @@ func NewPVCDefinition(pvcName string, size string, annotations, labels map[strin
 	}
 }
 
+// NewBlockPVCDefinition creates a PVC definition with volumeMode 'Block'
+func NewBlockPVCDefinition(pvcName string, size string, annotations, labels map[string]string) *k8sv1.PersistentVolumeClaim {
+	pvcDef := NewPVCDefinition(pvcName, size, annotations, labels)
+	volumeMode := corev1.PersistentVolumeMode(corev1.PersistentVolumeBlock)
```

Comment:
You shouldn't need this cast, since corev1.PersistentVolumeBlock is already a PersistentVolumeMode